apps/pie-storybook/stories/pie-modal.stories.ts

```diff
@@ -136,7 +144,8 @@ const BaseStoryTemplate = (props: ModalProps): TemplateResult => {
             returnFocusAfterCloseSelector="${ifDefined(returnFocusAfterCloseSelector)}"
             ?isOpen="${isOpen}"
             dir="${dir}"
-            size="${size}">
+            size="${size}"
+            .leadingAction="${leadingAction}">
```

Comment:
It will bind the to the prop leadingAction to itself. There's a bit here about it: https://lit.dev/docs/v1/components/templates/#bind-properties-to-templated-elements